src/status_im2/contexts/communities/home/view.cljs

```diff
@@ -8,12 +8,24 @@
             [quo2.foundations.colors :as colors]
             [quo2.components.community.discover-card :as discover-card]
             [quo2.components.navigation.top-nav :as topnav]
+            [status-im2.setup.config :as config]
             [status-im2.common.plus-button.view :as components.plus-button]
             [utils.re-frame :as rf]
             [i18n.i18n :as i18n]
             [quo2.core :as quo]
             [status-im.ui.screens.communities.community-options-bottom-sheet :as home-actions]))
 
+(defn navigate-to-activity-center []
+  (rf/dispatch [:mark-all-activity-center-notifications-as-read])
+  (if config/new-activity-center-enabled?
+    (rf/dispatch [:show-popover {:view                        :activity-center
```

Comment:
What do you think of making this dispatch to show the activity center reused in `screens.chat.home/home-tooltip-view`? It seems both will always need to be in sync, but with the duplication someone in the future might forget to update one of them.